index.bs

```diff
@@ -2950,13 +2968,19 @@ The following abstract operations support the implementation of the
 <div algorithm>
  <dfn abstract-op lt="ReadableStreamDefaultControllerEnqueue"
  id="readable-stream-default-controller-enqueue">ReadableStreamDefaultControllerEnqueue(|controller|,
- |chunk|)</dfn> performs the following steps:
+ |chunk|, |transferList|)</dfn> performs the following steps:
 
  1. If ! [$ReadableStreamDefaultControllerCanCloseOrEnqueue$](|controller|) is false, return.
```

Comment:
Generally when the public API has a precondition, we put an assert in the private API to enforce the precondition (to provide a hint to implementers and other specs). Something like
> Assert: |transferList| is empty or |controller|.[[isOwning]] is true.

Reply:
Added